src/horizontal_series.rs

```diff
@@ -119,17 +120,26 @@ impl HorizontalSeries {
                         svg_elements.push(
                             html!(<polyline points={points} class={classes.to_owned()} fill="none"/>),
                         );
-                    } else {
+                    } else if props.series_type == SeriesType::Bar {
```

Comment:
Cargo clippy should probably highlight that we need a match statement now…

Reply:
Updated to use match statements instead. (Though, Cargo clippy didn't highlight the need for a match; Rory also found this on the code.)